### DISCUSSION TITLE
Where are these trust settings saved?

### DISCUSSION THREAD

**Discussion by GeriZ** [dbeaver/dbeaver#35542] [Q&A] [answered]
### Description

I've activated the "trust all content" settings in Artifacts and Authorities under Preferences - Install/Update - Trust. Then I've copied the whole directory C:\Users\xxxx\AppData\Roaming\DBeaverData to a backup location. Some times later I've deleted C:\Program Files\DBeaver and C:\Users\xxxx\AppData\Roaming\DBeaverData, reinstalled DBeaver and copied C:\Users\xxxx\AppData\Roaming\DBeaverData back from the backup location to have all my settings restored.

Interestingly both trust settings are not activated now. Where are these settings saved?

### DBeaver Version

Community Edition 24.1.1

### Operating System

Windows 10

### Database and driver

DB2

**Comment by GeriZ** [comment]
Any idea from one of the DBeaver developers?

**Comment by GeriZ** [comment]
It seems that these settings are saved in org.eclipse.equinox.p2.engine.prefs under C:\Program Files\DBeaver\p2\org.eclipse.equinox.p2.engine\profileRegistry\DefaultProfile.profile\.data\.settings.

Should user preferences not be saved anywhere in C:\Users\xxxx\AppData\Roaming\DBeaverData?

**Comment by serjiokov** [comment, marked as answer]
Hi @GeriZ,
We are using:
```
1 Configuration scope. Preferences stored in this scope are shared by all workspaces that are launched using a particular configuration of Eclipse plug-ins. On a single-user installation, this serves to capture preferences that are common to all workspaces launched by that user. On a multi-user installation, these preferences are shared by all users of the configuration.
2 Instance scope. Preferences in this scope are specific to a single Eclipse workspace. The old API method getPluginPreferences on Plugin stores its preferences at this scope.
```
The location of preferences depends on the installation mode (for All users, Only for me) or unzip mode.
By default, you can find your preferences at   
C:\Users\xxxx\AppData\Roaming\DBeaverData\workspace6\.metadata\.plugins
If you are using unzip to some location the preferences path will be relative to your product location.
